crates/polars-core/src/chunked_array/ops/explode_and_offsets.rs

```diff
@@ -14,10 +15,13 @@ impl ChunkExplode for ListChunked {
         Ok(offsets)
     }
 
+    /// Returns the underlying values buffer and the corresponding offsets.
+    ///
+    /// A list array's memory layout is actually already 'exploded', so we can just take the values
```

Comment:
I think this should remain a comment below. 

In the docstring we should document what we return, not per-se how we do that and why that is allowed.